jetty-server/src/test/java/org/eclipse/jetty/server/ConnectorTimeoutTest.java

```diff
@@ -766,8 +773,156 @@ public void testMaxIdleWithDelayedDispatch() throws Exception
         Assert.assertTrue(System.currentTimeMillis() - start > minimumTestRuntime);
         Assert.assertTrue(System.currentTimeMillis() - start < maximumTestRuntime);
     }
-    
 
+    @Test(timeout=60000)
+    public void testSlowClientRequestNoLimit() throws Exception
+    {
```

Comment:
yup - but don't think I'll get time to do that before I go on vacation.  Will see....